content/questions/foundations/question-104.md

```diff
@@ -0,0 +1,12 @@
+---
+archetype: "questions"
+title: "Question 104"
+question: "If a GitHub Discussion is converted into an Issue and the issue is closed or referred to using its corresponding # number, will the discussion be modified?"
+draft: false
+---
+
+
+> https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion
+1. [ ] Yes
+1. [x] No
+> When you "convert" a discussion into an Issue it essentially creates a new issue that is soft linked to the discussion and pre-fills the issue information. There isn't a bigger connection between them.
```

Comment:
Sorry - missed that one previously.

The explanations are shown only for wrong answers - at the end of the practice test. There is no use for explanations for correct answer - they will not be shown
```suggestion
1. [x] No
1. [ ] Yes
> When you "convert" a discussion into an Issue it essentially creates a new issue that is soft linked to the discussion and pre-fills the issue information. There isn't a bigger connection between them.
```